contracts/ClaimsManager.sol

```diff
@@ -378,18 +380,14 @@ contract ClaimsManager is
         );
         claim.status = ClaimStatus.SettlementAccepted;
         // If settlement amount in USD causes the policy coverage to be exceeded, clip the API3 amount being paid out
-        uint256 settlementAmountInApi3 = claimIndexToProposedSettlementAmountInApi3[
-                claimIndex
-            ];
         uint256 settlementAmountInUsd = claimIndexToProposedSettlementAmountInUsd[
                 claimIndex
             ];
         uint256 clippedAmountInUsd = updatePolicyCoverage(
             claim.policyHash,
             settlementAmountInUsd
         );
-        uint256 clippedAmountInApi3 = (settlementAmountInApi3 *
-            clippedAmountInUsd) / settlementAmountInUsd;
+        clippedAmountInApi3 = convertUsdToApi3(clippedAmountInUsd);
```

Comment:
So here is where the magic happens. Instead of using rule of 3 we convert to API3 based on the current price making this function statically return a better approximate of the future payout, right?

Reply:
I'm wondering how much time will pass between settlement and resolution but I guess [this issue](https://github.com/api3dao/claims-manager/issues/48) will handle a possible slippage

Reply:
The settlement is proposed in USD and it's converted to API3 at the moment this function is called. So the slippage I'm referring is for the 2 minutes between the claimant sending the transaction that calls `acceptSettlement()` and that transaction being confirmed.